src/gui/accountmanager.cpp

```diff
@@ -78,40 +77,45 @@ bool AccountManager::restore()
 bool AccountManager::restoreFromLegacySettings()
 {
     qCInfo(lcAccountManager) << "Migrate: restoreFromLegacySettings, checking settings group"
-             << Theme::instance()->appName();
+                             << Theme::instance()->appName();
```

Comment:
@ckamm Humm OK, and is there a way to just tell it not to worry about that?

Also maybe worth trying `DontAlign ` together with `BinPackArguments: false`.